.github/workflows/ci.yaml

```diff
@@ -13,15 +13,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.2.4
+
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "18.x"
-
-      - name: Setup pnpm
-        uses: pnpm/action-setup@v2.2.4
+          node-version-file: '.nvmrc'
+          cache: 'pnpm'
 
-      - run: pnpm install
+      - run: pnpm install --frozen-lockfile
```

Comment:
I hope this will work with Renovate - our automated dependency updates bot.

Reply:
`frozen-lockfile` here in ci is just not to touch the lock file after pnpm install, so it will remains the same as in the repo.

This not going to affect Renovate in anyway 🙇🏼

Reply:
ok, let's give it a try :)